File: 프로그래머스/lv0/120904. 숫자 찾기/숫자 찾기.js
function solution(num, k) {
    // 정수num -> 문자열배열num
    // 문자열배열num 에서 k와 같은 정수를 찾아 반환
    
    let str = num.toString().split("");
    
    let search = str.indexOf("" + k);
    
    if(search === -1) return -1;
    
    return search + 1;
} // end

File: 프로그래머스/lv0/120904. 숫자 찾기/숫자 찾기.java
class Solution {
    public int solution(int num, int k) {
        
        String str = Integer.toString(num);
        
        if(str.indexOf("" + k) == -1) return -1;
        
        return str.indexOf("" + k) + 1;
        
        
    }
} // end